force-app/main/default/permissionsets/FeatureC_User.permissionset-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PermissionSet xmlns="http://soap.sforce.com/2006/04/metadata">
+    <customPermissions>
+        <enabled>true</enabled>
+        <name>Feature_C</name>
+    </customPermissions>
+    <description>Grants assigned users access to &quot;FeatureC&quot;. Users must be assigned a &quot;Demo FeatureC&quot; Permission Set License before this permission set can be assigned to them.</description>
+    <hasActivationRequired>false</hasActivationRequired>
+    <label>FeatureC User</label>
```

Comment:
could we change all the labels for perm sets to "FeatureX Grant Access" and similar for file names?
The "User" piece doesn't add any value

Reply:
API names cannot be changed once they are released.